Rakefile

```diff
@@ -454,49 +454,27 @@ task :get_availability_shields  do
     numeric_version = (reldev == 'release') ? site_config['release_version'] : site_config['devel_version']
     meat_index_file = File.join("tmp", "build_dbs","#{reldev}-bioc.meat-index.txt")
     json_file = File.join("assets", "packages", "json", numeric_version, "bioc", "packages.json")
-    json_obj = JSON.parse(File.read(json_file))
+
     indexList=[]
     if File.exists? meat_index_file
+
       mitxt = File.readlines(meat_index_file).join
       meat_index = Dcf.parse(mitxt)
+      json_obj = JSON.parse(File.read(json_file))
+
       if (not meat_index.nil?)
-        for item in meat_index
-          pkg = item['Package']
-          unsupported = item['UnsupportedPlatforms']
-          status = item['PackageStatus']
-          if status == "Deprecated"
-            img = "none"
-          else
-            if unsupported == "None"
-              img = "all"
-            else
-              img = "some"
-            end
-          end
-          puts "Creating badge for #{pkg} :  #{img}"
-          srcdir = File.join('assets', 'images', 'shields', 'availability')
-          destdir = File.join('assets', 'shields', 'availability', numeric_version)
-          FileUtils.mkdir_p destdir
-          src = File.join(srcdir, "#{img}.svg")
-          dest = File.join(destdir, "#{pkg}.svg")
-          res = FileUtils.copy(src, dest)
-          puts("    copied #{src} to #{dest}")
-          indexList.push(pkg)
-        end
-        unknown = json_obj.keys.sort - indexList.sort
-        if unknown.length != 0
-          for item in unknown
-            puts "Creating badge for #{item} :  unknown"
-            srcdir = File.join('assets', 'images', 'shields', 'availability')
-            destdir = File.join('assets', 'shields', 'availability', numeric_version)
-            src = File.join(srcdir, "unknown-build.svg")
-            dest = File.join(destdir, "#{item}.svg")
-            res = FileUtils.copy(src, dest)
-            puts("    copied #{src} to #{dest}")
-          end
-        end
+	for item in meat_index
+	  get_availability(item, numeric_version)
+	  indexList.push(item['Package'])
+	end
+	unknown = json_obj.keys.sort - indexList.sort
+	if unknown.length != 0
+	  for item in unknown
+	    availabilityBadge(item, "unknown-build", numeric_version)
+	  end
+	end
       else
-        puts "ERROR meat_index nil"
+	puts "ERROR meat_index nil"
```

Comment:
sorry to be a pain but would rather not see the whitespace changes here and below...

Reply:
how do I change back to a whitespace-cleanup call without losing the changes?